src/app/pages/network/forms/interfaces-form.component.ts

```diff
@@ -561,9 +571,22 @@ export class InterfacesFormComponent extends ViewControllerComponent implements
         this.entityForm.setDisabled('lacpdu_rate', true, true);
         this.entityForm.setDisabled('xmit_hash_policy', true, true);
       }
+      if (value === LinkAggregationProtocol.Failover) {
+        this.lagPortsField.tooltip = helptext.lagg_interfaces_tooltip + ' ' + helptext.lagg_interfaces_failover_tooltip;
+      } else {
+        this.lagPortsField.tooltip = helptext.lagg_interfaces_tooltip;
+      }
     });
   }
 
+  updateSearchOptions(value = '', values?: string[]): void {
+    this.lagPortsField.searchOptions = this.lagPortsOption.filter((option) => (
+      (!value
```

Comment:
Break down these checks as separate variables so it's more readable what's being filtered?